specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1930,6 +2061,106 @@
         }
       },
       "description": "The backup key result, containing the backup blob."
+    },
+    "KeyRotationPolicy": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The key policy id.",
+          "readOnly": true
+        },
+        "lifetimeActions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LifetimeActions"
+          },
+          "description": "Actions that will be performed by Key Vault over the lifetime of a key."
+        },
+        "attributes": {
+          "$ref": "#/definitions/KeyRotationPolicyAttributes",
+          "description": "The key rotation policy attributes."
+        }
+      },
+      "description": "Management policy for a key."
+    },
+    "LifetimeActions": {
+      "properties": {
+        "trigger": {
+          "$ref": "#/definitions/LifetimeActionsTrigger",
+          "description": "The condition that will execute the action."
+        },
+        "action": {
+          "$ref": "#/definitions/LifetimeActionsType",
+          "description": "The action that will be executed."
+        }
+      },
+      "description": "Action and its trigger that will be performed by Key Vault over the lifetime of a key."
+    },
+    "LifetimeActionsTrigger": {
+      "properties": {
+        "timeAfterCreation": {
+          "type": "string",
+          "description": "Days after creation to attempt rotate."
+        },
+        "timeBeforeExpiry": {
+          "type": "string",
+          "description": "Days before expiry to attempt rotate."
+        }
+      },
+      "description": "A condition to be satisfied for an action to be executed."
+    },
+    "LifetimeActionsType": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the action.",
+          "enum": [
+            "Rotate",
+            "Notify"
+          ],
+          "x-ms-enum": {
+            "name": "ActionType",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "Rotate",
+                "description": "Rotate the key based on the key policy."
+              },
+              {
+                "value": "Notify",
+                "description": "Trigger event grid events."
+              }
+            ]
+          }
+        }
+      },
+      "description": "The action that will be executed."
+    },
+    "KeyRotationPolicyAttributes": {
+      "properties": {
+        "rotationEnabled": {
+          "type": "boolean",
+          "default": false,
+          "description": "Determines whether key auto rotation is enabled."
+        },
+        "rotationTime": {
+          "type": "string",
+          "description": "Time to attempt rotate. Format examples: 90 days: 90D, 3 months: 3M, 48 hours: 48H, 1 year and 10 days: 1Y10D"
```

Comment:
Could 3M be misinterpreted as 3 minutes instead of 3 months?

Reply:
The examples need to be updated to ISO 8601. It must be `P3M` which is different than `PT3M`.

Reply:
this should have been updated